UKHomeOffice/nmsw-ui
# NMSW-1186 Review packages

# Ticket



----

## AC

Review all dependabot alerts and update packages

----

## To test

- remove node_modules
- run npm ci
- run eslint > linter should run
- run npm test > unit/integration tests should pass
- run npm build > app should build a production version
- run cypress tests > tests should run and pass

----

## Notes
